list.h

```diff
@@ -0,0 +1,252 @@
+#ifndef _LINUX_LIST_H
+#define _LINUX_LIST_H
```

Comment:
Is /usr/include/sys/queue.h ok to use in a apache 2 project? It has a list at least.